nixos/modules/misc/documentation.nix

```diff
@@ -327,12 +338,14 @@ in
       environment.systemPackages = []
         ++ optional cfg.man.enable manual.manpages
         ++ optionals cfg.doc.enable [ manual.manualHTML nixos-help ];
-
-      services.getty.helpLine = mkIf cfg.doc.enable (
-          "\nRun 'nixos-help' for the NixOS manual."
-      );
     })
 
+    (optionalAttrs (options?services.getty) (
```

Comment:
I don't like `option ? ` checks if they're avoidable. In this case, how about turning it around: Moving this definition into the getty module directly. Could also condition it on getty being enabled.

Reply:
I don't like it either actually, but I figured this implementation was less scope-creep-y. I'll move it.